doc/CDSL.md

```diff
@@ -1 +1,22 @@
 # CDSL
+CDSL is Component Definition Language used to create and define component in Robocomp. With CDSL you specify how the component will communicate with the world. 
+
+A basic cdsl file can be generated as given below. Go to a folder and type the following commands to make a directory and generate a component :
+https://robocomp.github.io/web/gsoc/2017/love/post5
```

Comment:
robocompdsl is not currently generating code in js.
It's not a good link for command examples.